src/ContentClient.ts

```diff
@@ -127,7 +127,7 @@ export class ContentClient implements ContentAPI {
                 const newName = name.endsWith('s') ? name.slice(0, -1) : name
                 let newValues: string[]
                 if (Array.isArray(value)) {
-                    newValues = value.map(element => `${element}`)
+                    newValues = [ ...value ]
```

Comment:
My concern here is that not all values will be strings. Should we convert it somewhere?